ardrone0_init:
  x: [1.15,0,0.95, 1,0, 0.0,0.0, 0.0,0.0]
  xdes: [1.15,0,0.95, 1,0, 0.0,0.0, 0.0,0.0]
  p: [-0.5092, 1.458, -1,1, -5, 1.3, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 0.3, 1.0] #without tracking position

constraint0_init:
  p: [1,0.17,0.5,1.5,1,1,1]

# the following are not implemented yet:
controller:
  HorizonDiskretization: 10
  HorizonLength: 1
  Tolerance: 1e-8
  UpdateIntervall: 0.01
  MaximumNumberofIterations: 10


